### ISSUE TITLE
InstructLab VM | Change required for the config.yaml file

### ISSUE THREAD

**Issue by Nirsisr** [instructlab/instructlab#2240]
When creating a new instructlab VM under redhat demo platform moving to the 'showroom' area there are bunch of instructions of how to get some basic experience with instructlab.

When I follow the instructions 1 by 1 the following error being raised:
**Screenshots**
**Device Info (please complete the following information):**
![image](https://github.com/user-attachments/assets/aff80069-e01a-44be-a1b0-8737c635c394)

**Expected behavior**: ilab data generate command should work properly after serving the dedicated model in other terminal without any user actions to be made.

**Additional context**
I was able to solve the bug by changing the config.yaml file as follows:
![image](https://github.com/user-attachments/assets/b9c8d0fd-8a72-4e40-9d96-89e82274de78)

Basically the value of model should be 'models/..' instead of 'instructlab/models/...', since the realtive path where we are running the ilab generate command from is the /instructlab folder as mention in your instructions.
Btw when trying changing the configuration file to '~/instructlab/models/...', in order to enforce the ilab generate to treat it as absolute path, same error being raised. 
So I guess that the ilab generate command is sensitive to relative paths and not supporting absolute path.
